source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -63,6 +63,49 @@ filter as a ``SingleResult`` type.
 To learn how to access data in a ``SingleResult`` see the :ref:`BSON
 <bson-unmarshalling>` guide.
 
+Modify Behavior
+~~~~~~~~~~~~~~~
+
+The ``Find()`` and ``FindOne()`` functions optionally take a
+``FindOptions`` and ``FindOneOptions`` type, which represents options
```

Comment:
You might want the word "respectively" there to show that each one applies to one of the aforementioned:
```suggestion
``FindOptions`` and ``FindOneOptions`` type respectively, which represents options
```